### PR TITLE
feature(playground): Migrate playground app to typescript

### PR DESCRIPTION
**Pull request by adarshaacharya**
- [x] add typescript setup for nextjs project
- [x] added types for packages installed
-  [x] convert .jsx based files to tsx and js to ts file

### CODE REVIEW

**Comment by adarshaacharya** [comment]
@shuding let me know if I can make any changes.

File: playground/pages/index.tsx [hunk 1]
```diff
@@ -40,9 +41,58 @@ const languageFontMap = {
...
+      style: 'normal',
+    },
+  ]
+}
+
+function withCache(fn: Function) {
   const cache = new Map()
-  return async (...args) => {
+  return async (...args: string[]) => {
+    console.log({ args })
```
Comment (shuding) [inline playground/pages/index.tsx:95]:
```suggestion
```

**Review by shuding — COMMENTED** [review]
Great work, thanks!

File: playground/pages/index.tsx (hunk 1 above)
Comment (adarshaacharya) [reply playground/pages/index.tsx]: sry, removed console.

**Pull request merged**